app/templates/search/macros/filters/subjects_tna.html

```diff
@@ -0,0 +1,57 @@
+{% from 'components/button/macro.html' import tnaButton %}
+{% from 'components/checkboxes/macro.html' import tnaCheckboxes %}
+
+{% macro subjects_tna(request, subjects_tna, number_of_subjects_to_show=10) %}
+  {% set is_tna_results_page = request.GET.get('group') in [None, 'tna'] %}
+  {% if is_tna_results_page %}
+    <div class="tna-aside tna-aside--tight tna-background-tint tna-!--margin-top-s">
+      {% set show_more_link = subjects_tna.items() | length > number_of_subjects_to_show %}
+      {% set subjects = [] %}
+      {% for subject_id, subject in subjects_tna.items() %}
+        {% if loop.index <= number_of_subjects_to_show %}
+          {% set subjects = subjects.append({
+          'text': subject,
+          'value': subject_id,
+          'checked': qs_is_value_active(request.GET, 'subjects', subject_id)
+        }) %}
+        {% elif loop.index > number_of_subjects_to_show %}
+          {% set show_more_link = true %}
+        {% endif %}
+      {% endfor %}
+      {{ tnaCheckboxes({
+            'label': 'Subjects',
+            'headingLevel': 3,
+            'headingSize': 'm',
+            'id': 'subjects',
+            'name': 'subjects',
+            'items': subjects,
+            'small': True,
+            'attributes': {
+              'form': 'search-form'
+            }
+          }) }}
+      <div class="tna-button-group tna-!--margin-top-s">
+        {{ tnaButton({
+              'text': 'Update',
+              'buttonElement': True,
+              'buttonType': 'submit',
+              'small': True,
+              'attributes': {
+                'form': 'search-form'
+              }
+            }) }}
+
+        {% set query_params = request.GET.copy().urlencode() %}
+
+        {% if show_more_link is true %}
+          {{ tnaButton({
+            'text': "See more",
+            'href': "/catalogue/search/?filter_list=subjects&" + query_params,
+            'plain': True,
+            'small': True
+          }) }}
+        {% endif %}
+      </div>
```

Comment:
You can make all buttons in the group small by default with an extra class:

```suggestion
      <div class="tna-button-group tna-button-group--small tna-!--margin-top-s">
        {{ tnaButton({
              'text': 'Update',
              'buttonElement': True,
              'buttonType': 'submit',
              'attributes': {
                'form': 'search-form'
              }
            }) }}

        {% set query_params = request.GET.copy().urlencode() %}

        {% if show_more_link is true %}
          {{ tnaButton({
            'text': "See more",
            'href': "/catalogue/search/?filter_list=subjects&" + query_params,
            'plain': True
          }) }}
        {% endif %}
      </div>
```

Reply:
Excellent - thanks @ahosgood, I hadn't realised that. I'll amend the commit and push 👍🏽